MdePkg/Include/Protocol/IncompatiblePciDeviceSupport.h

```diff
@@ -78,6 +78,15 @@
     0xeb23f55a, 0x7863, 0x4ac2, {0x8d, 0x3d, 0x95, 0x65, 0x35, 0xde, 0x03, 0x75} \
   }
 
+///
+/// Extension to Acpi10.h for describing the ROM BAR as the incompatible BAR.
+///
+/// Specifying the ROM bar as incompatible indicates to the PciBusDxe driver to
+/// ignore the ROM BAR altogether.  Currently, ACPI only defines 0x01-0x03 for
+/// MEM, IO, BUS.
+///
+#define INCOMPATIBLE_ACPI_ADDRESS_SPACE_TYPE_ROM  0x0F

```

Comment:
This has not been added to the UEFI PI Specifications for the Incompatible PCI Device Support Protocol.  Also, the latest ACPI Specification says that values 3..191 are reserved, which means this value of 0x0F could conflict with a future version of the ACPI Specification.  

This would require changes to both the ACPI Spec to allocate a value for this use case and an update to the UEFI PI Specification.  This PR should be dropped and changed to a code first spec/code change approach.  Unless an alternate method of indicating a PCI Option ROM BAR override is used.